v3/lints/cabf_smime_br/mailbox_validated_enforce_subject_field_restrictions.go

```diff
@@ -0,0 +1,94 @@
+package cabf_smime_br
+
+/*
+ * ZLint Copyright 2021 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import (
+	"fmt"
+
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/util"
+)
+
+// mailboxValidatedEnforceSubjectFieldRestrictions - linter to enforce MAY/SHALL NOT requirements for mailbox validated SMIME certificates
+type mailboxValidatedEnforceSubjectFieldRestrictions struct {
```

Comment:
This seems specifically be targeting the "strict" profile from the smime BRs. Do you think the lint name should reflect that?

Reply:
So, this lint is actually targeting all three of the levels of mailbox-validated SMIME lints.  The table in 7.1.4.2.3 has Legacy, Multipurpose and Strict mailbox-validated certs allowed the same fields